apps/gr_satellites

```diff
@@ -150,6 +155,7 @@ class gr_satellites_top_block(gr.top_block):
     def __init__(self, parser):
         gr.top_block.__init__(self, 'gr-satellites top block')
         sat = parse_satellite(sys.argv[1])
+        norad = yamlfiles.open_satyaml(**parse_satellite(sys.argv[1]))['norad']
```

Comment:
When testing this I have noticed that if we specify a satellite that doesn't exist, then we get a traceback:
```
$ gr_satellites foo
Traceback (most recent call last):
  File "/usr/local/bin/gr_satellites", line 333, in <module>
    main()
  File "/usr/local/bin/gr_satellites", line 318, in main
    tb = gr_satellites_top_block(parser)
  File "/usr/local/bin/gr_satellites", line 158, in __init__
    norad = yamlfiles.open_satyaml(**parse_satellite(sys.argv[1]))['norad']
  File "/usr/local/lib/python3.10/site-packages/satellites/satyaml/satyaml.py", line 221, in open_satyaml
    return self.search_name(name)
  File "/usr/local/lib/python3.10/site-packages/satellites/satyaml/satyaml.py", line 205, in search_name
    raise ValueError('satellite not found')
ValueError: satellite not found
```

In the main branch, the error is given directly, without a traceback:
```
$ gr_satellites foo
Satellite foo not found
```
It seems that we need to move some of the error handling to this point of the code.